react/src/index.js

```diff
@@ -46,7 +46,7 @@ if (window.location.hostname === "localhost") {
 
 let BACKEND_URL
 const DSN = process.env.REACT_APP_DSN
-const RELEASE = release() || process.env.REACT_APP_RELEASE
+const RELEASE = release("application.monitoring.javascript") || process.env.REACT_APP_RELEASE
```

Comment:
decided to pass the packageName from here, just so it's readable from src/index.js, which is a more commonly viewed file than release.js which has the `release()` function.

so it serves as a reminder that we're using packageName@version.